engine/swarm/secrets.md

```diff
@@ -338,16 +339,16 @@ generate the site key and certificate, name the files `site.key` and
     `site.conf`. You can store any file as a secret as long as it is smaller
     than 500 KB. This allows you to decouple the key, certificate, and
     configuration from the services that will use them. In each of these
-    commands, the `-f` flag takes the path to the file containing the secret
-    on the host machine's filesystem. In these examples, the secret name and
-    the file name are the same.
+    commands, the last argument represents the path to the file to read the
+    secret from on the host machine's filesystem. In these examples, the secret
+    name and the file name are the same.
 
     ```bash
-    $ docker secret create site.key -f site.key
+    $ docker secret create site.key site.key
```

Comment:
small nit: looks unclear which one is the name and which one is the actual file.